.github/workflows/riot-ryot.yml

```diff
@@ -0,0 +1,98 @@
+---
+# Run 'compile_and_test_for_board.py' on connected boards.
+#
+# This workflow will run on a RYOT machine and lunch all tests on all boards
+# currently connected to that machine.
+#
+# Documentation:
+#
+# * Setup a RYOT machine:
+#   https://github.com/fjmolinas/riot-ryot/blob/master/setup.md
+#
+# * Requirements (filled by a RYOT machine):
+#   * ssh access (required secrets set)
+#   * all required flashing tools installed
+#   * udev rules that map all BOARD to /dev/riot/tty-$(BOARD)
+#   * RIOT_MAKEFILES_GLOBAL_PRE that sets PORT and DEBUG_ADAPTER_ID for each
+#     BOARD http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#   * Static list of connected boards or make target listing them in json format
+#     so that fromJSON can be used.
+
+name: riot-ryot
+
+on:
+  # REMOVEME
+  pull_request:
+    branches:
+      - '*'
+  # Schedule weekly runs Saturday at 00:00 on master
+  schedule:
+    - cron: '00 0 * * 6'
+  push:
+    # Run on all new release candidates
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-RC[0-9]*'
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]'
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*'
+env:
+  # RIOT_MAKEFILES_GLOBAL_PRE to set for and DEBUG_ADAPTER_ID per BOARD
+  RIOT_MAKEFILES_GLOBAL_PRE: '/builds/conf/makefiles.pre'
+  UNIFLASH_PATH: '/opt/ti/uniflash'
```

Comment:
Can be removed, left over from when I was flashing over ssh.

Reply:
I was wrong, currently I set this in `etc/environment` but since a shell is not opened these are not loaded.

Reply:
Added a comment, let me know if that's OK.

Reply:
> I was wrong, currently I set this in `etc/environment` but since a shell is not opened these are not loaded.

That is surprising, `/etc/environment` should not require a shell, as it should be handled by `pam`. This is why its syntax does not allow shell substitution but only direct affectation. 
Unlike `/etc/profile` that is only run on some of the shells (so not always with non-interactive ssh).

But indeed the way they are started may be different than an ssh login.
There may be a need to allow passing the configuration to the systemd started runner.
Maybe it does not whitelist all variables or something.